style/tests.qmd

```diff
@@ -0,0 +1,68 @@
+---
+title: "Test style guide"
```

Comment:
```suggestion
title: "Writing Python tests"
```

Since it is in the style guide section, it doesn't need the style guide in the name.